pylibui/controls/button.py

```diff
@@ -22,7 +22,8 @@ def handler(window, data):
             self.onClick(data)
             return 0
 
-        self.clickHandler = libui.uiButtonOnClicked(self.control, handler, None)
+        self.clickHandler = libui.uiButtonOnClicked(self.control, handler,
+                                                    None)
```

Comment:
If we leave "None" on the same line, the line width is 85 characters; that's why I did this :-)